tests/ping.spec.ts

```diff
@@ -57,13 +62,51 @@ test(`Check ${basemapStylesUrl} availability`, async ({ request }) => {
   expect(responseObj.layers[0]).toBeDefined();
 });
 
-test(`Check ${atlasLayersUrl} availability`, async ({ request }) => {
-  expect(atlasLayersUrl).toBeDefined();
-  const response = await request.get(atlasLayersUrl!);
-  expect(response.status()).toEqual(200);
-  const responseObj = await response.json();
-  expect(responseObj[0]).toBeDefined();
-  expect(responseObj[0].id.length).toBeGreaterThan(0);
+languagesToTestLayers.forEach((language) => {
+  test(`Check ${atlasLayersUrl} to give correct language response (${language}) and style ninja json content to match ${language} locale`, async ({
+    request,
+  }) => {
+    expect(atlasLayersUrl).toBeDefined();
+    const response = await request.get(atlasLayersUrl!, {
```

Comment:
expect(atlasLayersUrl).toBeDefined(); stops code execution if it is not set

Reply:
in practise this error will never be thrown 
because if atlasLayersUrl is undefined it will fail with error (the test)
and this param is set at .env so  it can not be null, just undefined

Reply:
sure, I can set a globalSetup code for environment to check if all of that is set. Maybe next time, as current task is not about that. Set a task or issue please here on github to let me not forget that

Reply:
sure, thanks. have a great day